src/openklant/components/klantinteracties/api/serializers/internetaken.py

```diff
@@ -68,6 +70,23 @@ class Meta:
             },
         }
 
+    def validate(self, attrs):
+        status = attrs.get("status", None)
+        if not status and self.instance:
```

Comment:
Minor thing that is more _"proper"_ 
(I think you are checking if instance is not None) As far as I know, they can only be false if None.

```suggestion
        if status is None and self.instance is not None:
```